x/claim/keeper/claim_test.go

```diff
@@ -22,14 +22,14 @@ func TestGetClaimRecordForArkeo(t *testing.T) {
 		{
 			Chain:                  types.ARKEO,
 			Address:                addr1,
-			InitialClaimableAmount: sdk.NewCoins(sdk.NewInt64Coin(types.DefaultClaimDenom, 100)),
-			ActionCompleted:        []bool{false, false},
+			InitialClaimableAmount: sdk.NewInt64Coin(types.DefaultClaimDenom, 300),
+			ActionCompleted:        []bool{false, false, false},
 		},
 		{
 			Chain:                  types.ARKEO,
 			Address:                addr2,
-			InitialClaimableAmount: sdk.NewCoins(sdk.NewInt64Coin(types.DefaultClaimDenom, 200)),
-			ActionCompleted:        []bool{false, false},
+			InitialClaimableAmount: sdk.NewInt64Coin(types.DefaultClaimDenom, 600),
+			ActionCompleted:        []bool{false, false, false},
```

Comment:
Here , if we can add a few more cases , like when `ActionCompleted` is empty , or when `ActionCompleted` only 1 / 2 items in it.  make sure it still pass the test, and the function doesn't cause panic

Reply:
yup, this is a good test. will add.

Reply:
I added a guard to ensure that we cannot set a record without the correct number of items in the array, I then added tests to ensure that it works correctly to avoid setting them.  Let me know if you think this is sufficient.

Reply:
https://github.com/arkeonetwork/arkeo/pull/41/files#diff-7103aae3d6697d5e85af8e61d198a523f535578d293edbc33aaa373e401bba08R166